packages/firestore/src/local/memory_persistence.ts

```diff
@@ -49,17 +59,31 @@ export class MemoryPersistence implements Persistence {
    */
   private mutationQueues: { [user: string]: MutationQueue } = {};
   private remoteDocumentCache = new MemoryRemoteDocumentCache();
-  private queryCache = new MemoryQueryCache();
+  private readonly queryCache: MemoryQueryCache; // = new MemoryQueryCache();
+  private readonly listenSequence = new ListenSequence(0);
 
   private _started = false;
 
+  private _referenceDelegate: ReferenceDelegate | null;
+  get referenceDelegate(): ReferenceDelegate {
+    return this._referenceDelegate!;
+  }
+
+  static lruPersistence(clientId: ClientId): MemoryPersistence {
```

Comment:
These methods in `indexeddb_persistence` are prefixed with `create`, which I like a lot. Can you do the same here?

Reply:
Sure.